pymc_bart/pgbart.py

```diff
@@ -55,9 +110,9 @@ class PGBART(ArrayStepShared):
     def __init__(
         self,
         vars=None,  # pylint: disable=redefined-builtin
-        num_particles=20,
-        batch="auto",
-        model=None,
+        num_particles: int = 20,
+        batch: Union[str, int] = "auto",
```

Comment:
Maybe settle on a tuple of floats with a default value of (0.1, 0.1)? This will require changes here https://github.com/pymc-devs/pymc-bart/blob/7c1c5bb4bd75cf2ca1545fb99e2d05d98cb1e0f8/pymc_bart/pgbart.py#L174-L181

Reply:
done in https://github.com/pymc-devs/pymc-bart/pull/78/commits/b12feb528096f7fe097cbafe45f78669c4caddee